mm2src/mm2_bitcoin/keys/src/public.rs

```diff
@@ -80,6 +80,13 @@ impl Public {
             Public::Normal(_) => None,
         }
     }
+
+    pub fn to_secp256k1_pubkey(&self) -> Result<PublicKey, SecpError> {
+        match self {
+            Public::Compressed(public) => PublicKey::from_slice(&**public),
+            Public::Normal(public) => PublicKey::from_slice(&**public),
+        }
+    }
 }
```

Comment:
You have the same modification in a different PR too. https://github.com/KomodoPlatform/komodo-defi-framework/pull/2408

Reply:
will cover it in the conflict that will arise between them.